iOS ATS Check bug (IPA shared offline)

<!-- ## IMPORTANT -->
<!-- Issues are ONLY for reporting BUGS. For support, feature requests, questions, queries, and discussions use our slack channel for limited support. Join MobSF Slack channel: http://tiny.cc/mobsf
-->
<!-- If you see errors while running setup/run scripts, it is mostly because you haven't installed the required dependencies correctly. You will have to look into the errors and figure out what is causing them and solve them accordingly. Make sure you have installed all the required dependencies and their correct versions as per the latest documentation. If you still find yourself at a dead end, join MobSF Slack channel: http://tiny.cc/mobsf Please do not post support/help request in GitHub issues, doing so will definitely waste our time get them closed without further response. -->

## ENVIRONMENT

```
OS and Version: OS Mojave
Python Version: 3.7
MobSF Version: 3.0.6
```

## EXPLANATION OF THE ISSUE

```
  __  __       _    ____  _____       _____  ___  
 |  \/  | ___ | |__/ ___||  ___|_   _|___ / / _ \ 
 | |\/| |/ _ \| '_ \___ \| |_  \ \ / / |_ \| | | |
 | |  | | (_) | |_) |__) |  _|  \ V / ___) | |_| |
 |_|  |_|\___/|_.__/____/|_|     \_/ |____(_)___/

[INFO] 02/Apr/2020 15:43:11 - Mobile Security Framework v3.0.6 Beta
REST API Key: 618941ebdec42a7f519ed5dca40d9b3c5f17606ca6142fc2fc725f36cce06c40
[INFO] 02/Apr/2020 15:43:11 - OS: Linux
[INFO] 02/Apr/2020 15:43:11 - Platform: Linux-4.4.0-173-generic-x86_64-with-Ubuntu-18.04-bionic
[INFO] 02/Apr/2020 15:43:11 - Dist: ubuntu 18.04 bionic
[INFO] 02/Apr/2020 15:43:11 - MobSF Basic Environment Check
[INFO] 02/Apr/2020 15:43:11 - Checking for Update.
[INFO] 02/Apr/2020 15:43:12 - No updates available.
[INFO] 02/Apr/2020 15:44:21 - MIME Type: application/octet-stream FILE: ziaanalytics.ipa
[INFO] 02/Apr/2020 15:44:22 - Performing Static Analysis of iOS IPA
[INFO] 02/Apr/2020 15:44:22 - iOS Static Analysis Started
[INFO] 02/Apr/2020 15:44:22 - iOS Binary (IPA) Analysis Started
[INFO] 02/Apr/2020 15:44:22 - Generating Hashes
[INFO] 02/Apr/2020 15:44:22 - Extracting IPA
[INFO] 02/Apr/2020 15:44:22 - Unzipping
[INFO] 02/Apr/2020 15:44:25 - Get Files, BIN Plist -> XML, and Normalize
[INFO] 02/Apr/2020 15:44:25 - iOS Info.plist Analysis Started
[INFO] 02/Apr/2020 15:44:25 - Finding Info.plist in iOS Binary
[INFO] 02/Apr/2020 15:44:25 - Checking Permissions
[INFO] 02/Apr/2020 15:44:25 - Checking for Insecure Connections
[ERROR] 02/Apr/2020 15:44:25 - Reading from Info.plist
Traceback (most recent call last):
  File "/root/Mobile-Security-Framework-MobSF/StaticAnalyzer/views/ios/plist_analysis.py", line 99, in plist_analysis
    plist_info['inseccon'] = check_transport_security(plist_obj)
  File "/root/Mobile-Security-Framework-MobSF/StaticAnalyzer/views/ios/app_transport_security.py", line 68, in check_transport_security
    if (config.get('NSExceptionAllowsInsecureHTTPLoads', False)
AttributeError: 'str' object has no attribute 'get'
[ERROR] 02/Apr/2020 15:44:25 - Error Perfroming Static Analysis
Traceback (most recent call last):
  File "/root/Mobile-Security-Framework-MobSF/StaticAnalyzer/views/ios/static_analyzer.py", line 99, in static_analyzer_ios
    app_dict['appstore'] = app_search(infoplist_dict.get('id'))
AttributeError: 'NoneType' object has no attribute 'get'
[ERROR] 02/Apr/2020 15:44:25 - 'NoneType' object has no attribute 'get'
[ERROR] 02/Apr/2020 15:44:25 - Internal Server Error: /StaticAnalyzer_iOS/
```
<!-- If you see errors while running setup/run scripts, join MobSF Slack channel: http://tiny.cc/mobsf to get limited support. -->

## STEPS TO REPRODUCE THE ISSUE

```
1. Upload the shared IPA
```


## LOG FILE

```
Paste the contents of logs/debug.log here or attach the log file.
```

<!--
BEFORE POSTING YOUR ISSUE/BUG
- These comments won't show up when you submit the issue.
- GitHub issues ARE NOT FOR FEATURE REQUESTS, SUPPORT, DISCUSSIONS AND QUESTIONS! 
- If you have questions, use our slack channel. Join MobSF Slack channel: http://tiny.cc/mobsf
- Reproduce issue in the latest master and try to add as much detail as possible.
- Search this repository (top of the page) for the issue and it has not been fixed or reported already.
- Once you open a bug, you should also provide additional information if requested. 
- Failure to do so in 25 days will result in closure of the bug without further communication.
-->

--- a/StaticAnalyzer/views/ios/app_transport_security.py
+++ b/StaticAnalyzer/views/ios/app_transport_security.py
@@ -63,7 +63,14 @@ def check_transport_security(p_list):
 
         exception_domains = ats_dict.get('NSExceptionDomains')
         if exception_domains:
+            ats.append({
+                'issue': 'NSExceptionDomains',
+                'status': 'info',
+                'description': ', '.join(exception_domains.keys()),
+            })
             for domain, config in exception_domains.items():
+                if not isinstance(config, dict):
+                    continue
                 old_exp = 'NSTemporaryExceptionAllowsInsecureHTTPLoads'
                 if (config.get('NSExceptionAllowsInsecureHTTPLoads', False)
                         or config.get(old_exp, False)):

